server/routes/notificationRouter.test.ts

```diff
@@ -0,0 +1,46 @@
+import type { Express } from 'express'
+import request from 'supertest'
+
+import { appWithAllRoutes } from './testutils/appSetup'
+import UserService from '../services/userService'
+
+jest.mock('../services/userService')
+
+const notificationId = 'test-notification-id'
+const expectedCookieName = `notification-banner-${notificationId}`
+const expectedMaxAgeSecs = 52 * 24 * 3600
```

Comment:
why 52 days?

Reply:
Sorry, got this wrong, was meant to be 52 _weeks_, I'll fix.

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/362/commits/62c47043f7c1e9951f05992842573f1ba19cac62